cocotb/__init__.py

```diff
@@ -164,12 +164,12 @@ def _initialise_testbench(root_name):
     test_str = os.getenv('TESTCASE')
     hooks_str = os.getenv('COCOTB_HOOKS', '')
 
-    if not module_str:
+    if module_str is None:
         raise ImportError("Environment variables defining the module(s) to " +
                           "execute not defined.  MODULE=\"%s\"" % (module_str))
```

Comment:
This is dumb, if the module string was not defined then there's no poitn printing it, especially if we're going to imply it was set to the string `None`.
```suggestion
        raise ImportError("Environment variables MODULE which defines the module(s) to 
                          "execute is not defined.")
```

Reply:
I feel like `ImportError` is also improper, will change to `ValueError`.

Reply:
Hadn't even noticed that, good idea.